Dockerfile

```diff
@@ -67,12 +67,12 @@ WORKDIR $GOPATH/src/github.com/go-swagger/go-swagger
 # Install go-swagger for validating swagger.yaml
 # This is https://github.com/kolyshkin/go-swagger/tree/golang-1.13-fix
 # TODO: move to under moby/ or fix upstream go-swagger to work for us.
-ENV GO_SWAGGER_COMMIT 5793aa66d4b4112c2602c716516e24710e4adbb5
+ENV GO_SWAGGER_COMMIT 306d9e849ecc68ee5bfb12437e507a2feee8031b
```

Comment:
Switching to https://github.com/kolyshkin/go-swagger/pull/1 to see if the fix works